src/toir.d

```diff
@@ -415,6 +415,7 @@ int intrinsic_op(FuncDeclaration fd)
         "4math6rndtolFNaNbNiNfeZl",
         "4math6yl2xp1FNaNbNiNfeeZe",
 
+        "4simd10__prefetchFNaNbNiNfxPvhZv",
```

Comment:
Because the template calls `__prefetch()`. I prefer to not have templates themselves be intrinsics, I'd rather they be straightforward functions.